### ISSUE TITLE
代码问题

### ISSUE THREAD

**Issue by JusC-C** [XiaojuanTang/RulE#6]
您好，非常感谢您出色的工作！我现在直接运行您的代码，跑出来的是RulE (emb.)的结果，请问我应该如何修改代码跑出RulE (emb & rule.)的结果呢？

**Comment by JusC-C** [comment]
logits, mask = model(all_h, all_r, None)
kge_score = model.compute_g_KGE(all_h,all_r)
logits = logits + alpha * kge_score
请问这个是否就表示使用了emb & rule？

**Comment by XiaojuanTang** [comment]
> logits, mask = model(all_h, all_r, None) kge_score = model.compute_g_KGE(all_h,all_r) logits = logits + alpha * kge_score 请问这个是否就表示使用了emb & rule？

Yes, that’s correct. The kge_score represents the score obtained using the KGE method, while the logits are obtained after applying rules.

**Comment by JusC-C** [comment]
> > logits, mask = model(all_h, all_r, None) kge_score = model.compute_g_KGE(all_h,all_r) logits = logits + alpha * kge_score 请问这个是否就表示使用了emb & rule？
> 
> Yes, that’s correct. The kge_score represents the score obtained using the KGE method, while the logits are obtained after applying rules.

Thank you for your reply! But is it a coincidence that the result of running your code now is almost the same as that of RulE (emb.)? Is the accuracy error of this result normal? We look forward to your reply!

**Comment by LHS23** [comment]
请问一下您得到RulE (emb & rule.)的结果了吗？我这儿一直都只有RulE (emb.)的结果。

**Comment by XiaojuanTang** [comment]
#8 Please see this issue.
